### ISSUE TITLE
Tag picture invalid

### ISSUE THREAD

**Issue by Tuckie** [gjtorikian/html-proofer#298]
Any idea why this throws a `Tag picture invalid` error? I'm running 2.6.1. It checks out fine with W3C's validator.

```html
<a class="first-banner" href="news/promo/">
<picture>
<!--[if IE 9]><video style="display: none;"><![endif]-->
<source srcset="assets/images/banner-large.png" media="(min-width: 1200px)" />
<!--[if IE 9]></video><![endif]-->
<img src="assets/images/banner-small.png" srcset="assets/images/banner-small.png" alt="the more you save" />
</picture>
</a>
```

**Comment by gjtorikian** [comment]
Hmm. Could you post the full error (including line number)?

**Comment by Tuckie** [comment]
I've created a reduced test case:
```html
<!DOCTYPE html>
<html lang="en">
  <head>
    <meta charset="utf-8">
    <title>title</title>
  </head>
  <body>
<picture>
<!--[if IE 9]><video style="display: none;"><![endif]-->
<source srcset="https://placehold.it/350x150" media="(min-width: 1200px)" />
<!--[if IE 9]></video><![endif]-->
<img src="https://placehold.it/350x150" alt="the more you save" />
</picture>
  </body>
</html>
```

```
Running ["ScriptCheck", "HtmlCheck", "ImageCheck", "LinkCheck"] on ./build on *.


Ran on 177 files!


- ./build/index.html
  *  Tag picture invalid (line 8)
rake aborted!
```

FYI, I'm currently running it with:

```ruby
task :proof do
  options = {
    check_external_hash: true,
    check_html: true,
    disable_external: true,
    typhoeus: { proxy: ENV['http_proxy'] }
  }
  HTML::Proofer.new('./build', options).run
end
```

**Comment by gjtorikian** [comment]
This'll be fixed in HTML-Proofer 3.0.

**Comment by Tuckie** [comment]
Awesome, thank you!

**Comment by gjtorikian** [comment]
Closed by HTMLProofer 3.0: https://github.com/gjtorikian/html-proofer/releases/tag/v3.0.0

**Comment by Tuckie** [comment]
It's working great, thanks!
